src/welcome/WelcomePage.jsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Helmet } from 'react-helmet';
+
+import { getConfig } from '@edx/frontend-platform';
+import { AppContext } from '@edx/frontend-platform/react';
+import { sendPageEvent } from '@edx/frontend-platform/analytics';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Form,
+  StatefulButton,
+  Hyperlink,
+} from '@edx/paragon';
+
+import messages from './messages';
+import { EDUCATION_LEVELS, GENDER_OPTIONS, YEAR_OF_BIRTH_OPTIONS } from './data/constants';
+import { registrationRequestSelector } from './data/selectors';
+
+import {
+  AuthnValidationFormGroup, RedirectLogistration,
+} from '../common-components';
+import { DEFAULT_REDIRECT_URL } from '../data/constants';
+
+const WelcomePage = (props) => {
+  sendPageEvent('login_and_registration', 'reset');
+  const { intl, registrationResult } = props;
+
+  const { authenticatedUser } = useContext(AppContext);
+  const [values, setValues] = useState({});
+  const [status, setStatus] = useState(null);
+  const [redirectUrl, setRedirectUrl] = useState(null);
+
+  const getOptions = () => ({
+    yearOfBirthOptions: [{
+      value: '',
+      label: intl.formatMessage(messages['registration.year.of.birth.label']),
+    }].concat(YEAR_OF_BIRTH_OPTIONS),
+    educationLevelOptions: EDUCATION_LEVELS.map(key => ({
+      value: key,
+      label: intl.formatMessage(messages[`registration.field.education.levels.${key || 'label'}`]),
+    })),
+    genderOptions: GENDER_OPTIONS.map(key => ({
+      value: key,
+      label: intl.formatMessage(messages[`registration.field.gender.options.${key || 'label'}`]),
+    })),
+  });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setStatus(registrationResult.success);
+    setRedirectUrl(registrationResult.redirectUrl);
+  };
+
+  const handleSkip = (e) => {
+    e.preventDefault();
+    setRedirectUrl(DEFAULT_REDIRECT_URL);
```

Comment:
You need to prepend the `LMS_BASE_URL` with the default URL as well otherwise, it will try to find it in the MFE routes and leads to a 404 page not found.